src/mainwindow.cpp

```diff
@@ -935,8 +935,10 @@ void MainWindow::fullScreenshot()
     sendNotify(m_saveIndex, m_saveFileName, r);
 }
 
-void MainWindow::savePath(const QString &path)
+void MainWindow::savePath(const QString &path, bool noNotify)
 {
+    m_noNotify = noNotify;
```

Comment:
Is this really necessary?

Reply:
Either that or you pass it to saveSpecificedPath as an additional argument.
I found this way more consistent with how it is done in noNotify() and in my opinion
the class variable that is supposed to control that no notifications are sent should probably
be set correctly when not sending a notification.
But i guess it doesn't really matter that much.

Reply:
I mean, `savePath` is the end of the program, you can just use the function parameter without setting `m_noNotify` here or just use `m_noNotify` with it set before savePath called, what do you think?